src/xcode/ENA/ENA/Source/Workers/Store/Store.swift

```diff
@@ -185,6 +185,9 @@ protocol HealthCertificateStoring: AnyObject {
 
 	var unseenTestCertificateCount: Int { get set }
 
+	var lastSelectedValidationCountry: Country { get set }
```

Comment:
perhaps belongs this to Protocol `HealthCertificateValidationCaching`? Or rename it and then group everything of the validation together.

Reply:
Would not be a good fit in my opinion. It does not belong to the cache, its not delivered by the server.